Shopify.md

```diff
@@ -0,0 +1,19 @@
+# BTCPay Server and Shopify Integration
+
+To integrate BTCPay Server into Shopify, you need to be running BTCPay Server v1.0.5.5 or newer.
+
+* In your BTCPay Server Store > Pay button enable `Allow anyone to create invoice`
+* In Shopify Settings > Payment Providers > Manual Payment Methods add one which contains "Bitcoin with BTCPayServer"
+* In Shopify Settings > Checkout > Additional Scripts input the following script, with the details from your BTCPayServer instead of the placeholder values.
+
+<script>
+    const BTCPAYSERVER_URL = "FULL_BTCPAYSERVER_URL_WITH_HTTPS";
+    const STORE_ID = "YOUR_BTCPAY_STORE_ID";
+</script>
+<script src="FULL_BTCPAYSERVER_URL_WITH_HTTPS/bundles/shopify-bundle.min.js"></script>
+
+To integrate registering of BtcPayServer transactions on Shopify:
+
+ * Create private app
+ * Copy-paste credentials from Shopify over to Integrations view in BtcPayServer
```

Comment:
There is now a little prompt that you click 
![image](https://user-images.githubusercontent.com/1818366/93618886-2a878c80-f9d8-11ea-82b5-52c880f2388c.png)

![image](https://user-images.githubusercontent.com/1818366/93618774-075cdd00-f9d8-11ea-8178-1c5185c28763.png)
![image](https://user-images.githubusercontent.com/1818366/93618927-3bd09900-f9d8-11ea-93ae-2226aa77a26c.png)

Reply:
@Kukks  Sweet changes, is there any way we can do better UX then that `click here and paste`, not a proper PR to comment on it, but since you already posted a photo, just wanted to ask. I think a button may be a better UX then this.

Reply:
Here is new flow... took out manual set of 3 fields, just using that flow with full URL... will be more intuitive for users.

![new flow](https://user-images.githubusercontent.com/5191402/94306187-1bed2800-ff38-11ea-9eff-a93ef985ffe6.gif)